library/mageia

```diff
@@ -1,5 +1,5 @@
 # maintainer: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
 
-latest: git://github.com/juanluisbaptiste/docker-brew-mageia@ed9534c591e9b03593e8111990395f77bbd256c0 4
-cauldron: git://github.com/juanluisbaptiste/docker-brew-mageia@ed9534c591e9b03593e8111990395f77bbd256c0 cauldron
-4: git://github.com/juanluisbaptiste/docker-brew-mageia@ed9534c591e9b03593e8111990395f77bbd256c0 4
+latest: git://github.com/juanluisbaptiste/docker-brew-mageia@a814b1bd92e4bd4967ca17c56cd873df67ac4d91 4 
```

Comment:
Your repo looks correct now, but I think Travis is failing on the extra blank space at the end of this line.

Also, shouldn't you be adding `5` in here somewhere? (and should it be the new "`latest`"?)